policies/submissions.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Submitting Your Project To CODEVID-19
+---
+
+Read these carefully, otherwise we can't consider your team for prizing!
+
+### Deliverables
+
+1. REQUIRED: A link to your working app (whether that's a FindCollabs link, the website URL, or a TestFlight/Google Play opt-in URL).
```

Comment:
Hard to say without a Markdown preview of it, but would bolding "REQUIRED" instead of all-caps be more readable?

Reply:
Sure. I actually just ran out of time on this, but that's what I should do.

BTW if you want to see the whole file in a preview you can either click the button at the top-right of the diff to switch to rich diff mode, or open the menu at the top right of the file in the change set to "View" which will open the full file at that commit. You can then preview the entire file if that's easier to read.